metricbeat/docs/metricbeat-options.asciidoc

```diff
@@ -200,3 +200,19 @@ A list of processors to apply to the data generated by the metricset.
 See <<filtering-and-enhancing-data>> for information about specifying
 processors in your config.
 
+[float]
+[[module-http-config-options]]
+=== Standard HTTP config options
```

Comment:
I wonder if we should just copy SSL settings here too, as both things are supported by the same helper

Reply:
I'm curious what @dedemorton thinks of this. Perhaps a fragment that we include in both (or multiple) places?

Reply:
@tsg Thank you for doing this! Right now, I agree that the user experience is not ideal (there's a lot of jumping around to find the full list of config options). Rather than pointing to another topic, I'd prefer to use an include to pull in any common (shared) options. To make this work, we'll need to add an attribute to the docs for each module (something like `:modulename: modulename`) and then use that attribute in the ID for shared sections. Do you want me to do that? This has been low down on my to-do list for awhile; I can bump the priority.

Reply:
...and looking at my notes, I see that the current design (pointing off to options) has resulted in problems for users in other areas (like not knowing that processors are valid in the config).

Reply:
I have this as a to-do item in a meta issue, but I've created a real issue to track the work: https://github.com/elastic/beats/issues/7160